flink-python/pyflink/fn_execution/coder_impl.py

```diff
@@ -93,6 +93,30 @@ def __repr__(self):
         return 'RowCoderImpl[%s]' % ', '.join(str(c) for c in self._field_coders)
 
 
+class TableFunctionRowCoderImpl(StreamCoderImpl):
+
+    def __init__(self, row_coder):
+        self._row_coder = row_coder
+        self._field_count = row_coder._filed_count
+
+    def encode_to_stream(self, value, out_stream, nested):
+        if value is None:
+            self.write_finish_message(out_stream)
```

Comment:
Avoid extra function call here(to avoid performance regression)? i.e.,  do `out_stream.write_byte(0x00)` directly here.